Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -21,29 +22,54 @@ final class SitterProfileViewModel: ObservableObject {
     /// The price per hour for walking charged by the pet sitter.
     @Published var pricePerHour: String = ""
 
+    /// Indicating whether an error has occurred during the network operation.
+    @Published var isErrorOccurred: Bool = false
+
+    /// Detailed error information for the user.
+    @Published var errorMessage: String = ""
```

Comment:
It could be easier, only manage the `errorMessage` property state. You can set `errorMessage` to `""` in the `setInitialValues` function.
WDYT?
```suggestion
    private(set) var errorMessage = "" {
        didSet {
            isErrorOccurred = !errorMessage.isEmpty
        }
    }
```

Reply:
Fixed with 7282f12